### PR TITLE
docs(simbridge): Troubleshoot - FlyPad Settings

### LINKED ISSUE #678: Update SimBridge Troubleshooting Guide
> Can I suggest a really small addition to web mcdu/remote display docs in general. A quick simbridge connectivity troubleshooting note. 
> 1) check simbridge running
> 2) check flypad if it is connected (wifi icon), if not how to resolve it (auto off and auto again)

As left by: Oyu#1741 who is willing to work on updating documentation.

### PR DESCRIPTION
**Pull request by Oyu07**
## Summary

Fixes https://github.com/flybywiresim/docs/issues/678
Adds images and guide on how to attempt a new connection to simbridge if it does not connect. "Sim Options - Simbridge Connection - Auto - off - Auto again
### Location

https://docs-git-fork-oyu-de-flypad-s-flybywire.vercel.app/simbridge/troubleshooting/#flypad-settings
docs\simbridge\troubleshooting.md

Discord username (if different from GitHub): Oyu#1741

### CODE REVIEW

**Comment by Oyu07** [comment]
made it as draft rn, just wanna get initial review before making it an actual PR.
I know it says mcdu server, thats existing flypados3 image

**Comment by Valastiri** [comment]
Suggest highlighting relevant areas with a red box to show user what to click on along side the text. 

An example of the crossed out wifi could be handy as well.

File: docs/simbridge/troubleshooting.md [hunk 1]
```diff
@@ -45,6 +45,19 @@ Steps to regain functionality:
...
+
+FlyPad will stop trying to connect to simbridge after a 5 minute timer.
+Check the connectivity status on FlyPad by checking if the wifi icon on the top is crossed.
+![FlyPad Wifi](../fbw-a32nx/assets/flypados3/flypad-dashboard-menu.png){loading=lazy}
+
+!!! block ""
+    ![FlyPad Sim Options](../fbw-a32nx/assets/flypados3/flypad-settings-sim-options.png){loading=lazy align=left width=50%}
+    
+    - If the wifi / connectivity icon is crossed, head to Flypad settings --> Sim Options
+    - To attempt connection : Simbridge Connection: Turn "AUTO" to "OFF" and "AUTO" again.
```
Comment (frankkopp) [inline docs/simbridge/troubleshooting.md]:
```suggestion
    - To attempt connection : Simbridge Connection: Turn "AUTO" to "OFF" for a few seconds and "AUTO" again.
```

I have noticed if you do it too quickly it sometimes doesn't take.

**Review by frankkopp — APPROVED** [review]

File: docs/simbridge/troubleshooting.md (hunk 1 above)
Comment (Oyu07) [reply docs/simbridge/troubleshooting.md]: thx for the suggestion. added in https://github.com/flybywiresim/docs/pull/682/commits/5b36dc93da15e120a88306012f3fa283b11e6ef9

**Comment by Oyu07** [comment]
realised the "settings and then simbridge" is not the best screenshot/guide ever. will change it tomorrow to include both a
flypad settings -> sim options
simbridge "Auto" "Off" box

**Comment by frankkopp** [comment]
> realised the "settings and then simbridge" is not the best screenshot/guide ever. will change it tomorrow to include both a flypad settings -> sim options simbridge "Auto" "Off" box

Updated the settings page for SimOptions anyway: https://github.com/flybywiresim/docs/pull/683
You can use the image there (copy) and add your arrow - I also have the original if you need it. 
You need to resize it then to 826px wide and optimize it with https://www.websiteplanet.com/webtools/imagecompressor/

**Comment by Oyu07** [comment]
> > realised the "settings and then simbridge" is not the best screenshot/guide ever. will change it tomorrow to include both a flypad settings -> sim options simbridge "Auto" "Off" box
> 
> Updated the settings page for SimOptions anyway: #683 You can use the image there (copy) and add your arrow - I also have the original if you need it. You need to resize it then to 826px wide and optimize it with https://www.websiteplanet.com/webtools/imagecompressor/

ah brilliant, thanks for the heads up and the link :)

File: docs/simbridge/troubleshooting.md [hunk 2]
```diff
@@ -45,6 +45,25 @@ Steps to regain functionality:
...
+:-------------------------:|:-------------------------:
+![Wifi Crossed](../fbw-a32nx/assets/flypados3/flypad-status-crossed.png){loading=lazy}  |  ![Wifi Normal](../fbw-a32nx/assets/flypados3/flypad-status-normal.png){loading=lazy}
+
+- If the wifi / connectivity icon is crossed, head to Flypad Settings --> Sim Options 
+
+FlyPad Settings            |  Sim Options
+:-------------------------:|:-------------------------:
+![FlyPad Sim Settings](../fbw-a32nx/assets/flypados3/flypad-settings-overview_sim_options_marked.png){loading=lazy}  |      ![FlyPad Sim Options](../fbw-a32nx/assets/flypados3/flypad-settings-sim-options-marked.png){loading=lazy}
+
+- To attempt connection : SimBridge Connection: Turn "AUTO" to "OFF" for a few seconds and "AUTO" again.
```
Comment (Valastiri) [inline docs/simbridge/troubleshooting.md]: This looks a little weird to read. Maybe do this instead?
```suggestion
To attempt connection:

- SimBridge Connection: Turn "AUTO" to "OFF" for a few seconds.
- Select "AUTO" again.
```

File: docs/simbridge/troubleshooting.md [hunk 3]
```diff
@@ -45,6 +45,25 @@ Steps to regain functionality:
 
 SimBridge should now be able to be minimized to the tray appropriately.
 
+## FlyPad Settings
+
+FlyPad will stop trying to connect to SimBridge after a 5 minute timer.
+Check the connectivity status on FlyPad by checking if the wifi icon on the top is crossed.
```
Comment (Valastiri) [inline docs/simbridge/troubleshooting.md]:
```suggestion
Check the connectivity status on the flyPad by checking if the wifi icon on the top is crossed.
```
Using EFB works as well. Branding is "flyPad".

File: docs/simbridge/troubleshooting.md [hunk 4]
```diff
@@ -45,6 +45,25 @@ Steps to regain functionality:
 
 SimBridge should now be able to be minimized to the tray appropriately.
 
+## FlyPad Settings
+
+FlyPad will stop trying to connect to SimBridge after a 5 minute timer.
```
Comment (Valastiri) [inline docs/simbridge/troubleshooting.md]: EFB here if you choose to use EFB on the lines below.

**Review by Valastiri — CHANGES_REQUESTED** [review]
Unfortunately I can't see image sizes on mobile for some reason. I assume they are optimized can check later.

**Comment by Oyu07** [comment]
> Unfortunately I can't see image sizes on mobile for some reason. I assume they are optimized can check later.

np
also I just saw it, gotta update the PR so it fits with this https://github.com/flybywiresim/docs/pull/687
didnt realise it got merged

File: docs/simbridge/troubleshooting.md [hunk 5]
```diff
@@ -66,6 +66,27 @@ Steps to regain functionality:
 
 SimBridge should now be able to be minimized to the tray appropriately.
 
+## FlyPad Settings
+
+EFB will stop trying to connect to SimBridge after a 5 minute timer.
```
Comment (Valastiri) [inline docs/simbridge/troubleshooting.md]:
```suggestion
The EFB will stop trying to connect to SimBridge after a 5 minute timer.
```

File: docs/simbridge/troubleshooting.md [hunk 6]
```diff
@@ -66,6 +66,27 @@ Steps to regain functionality:
...
+Check the connectivity status on the EFB by checking if the wifi icon on the top is crossed.
+![FlyPad Wifi](../fbw-a32nx/assets/flypados3/flypad-dashboard-connectivity.png){loading=lazy}
+
+Icon Crossed             |  Icon Normal
+:-------------------------:|:-------------------------:
+![Wifi Crossed](../fbw-a32nx/assets/flypados3/flypad-status-crossed.png){loading=lazy}  |  ![Wifi Normal](../fbw-a32nx/assets/flypados3/flypad-status-normal.png){loading=lazy}
+
+- If the wifi / connectivity icon is crossed, head to the flypad Settings --> Sim Options
```
Comment (Valastiri) [inline docs/simbridge/troubleshooting.md]:
```suggestion
- If the wifi / connectivity icon is crossed, head to the flyPad Settings --> Sim Options 
```

**Review by Valastiri — APPROVED** [review]

**Comment by Valastiri** [comment]
check build

**Pull request merged**